packages/bpk-component-popover/readme.md

```diff
@@ -76,6 +84,7 @@ class App extends Component {
 | closeButtonIcon       | bool                          | false    | true          |
 | portalStyle           | object                        | false    | null          |
 | portalClassName       | string                        | false    | null          |
+| renderTarget          | func                          | false    | null          |
```

Comment:
Also this should be `oneOf([func, node])`

Reply:
right, I copy pasted from modal 😇